python/ray/train/xgboost/xgboost_trainer.py

```diff
@@ -28,22 +28,22 @@ class XGBoostTrainer(GBDTTrainer):
         `XGBoost documentation <https://xgboost.readthedocs.io>`__.
 
     Example:
-        .. code-block:: python
 
-            import ray
-
-            from ray.train.xgboost import XGBoostTrainer
-            from ray.air.config import ScalingConfig
-
-            train_dataset = ray.data.from_items(
-                [{"x": x, "y": x + 1} for x in range(32)])
-            trainer = XGBoostTrainer(
-                label_column="y",
-                params={"objective": "reg:squarederror"},
-                scaling_config=ScalingConfig(num_workers=3),
-                datasets={"train": train_dataset}
-            )
-            result = trainer.fit()
+    >>> import ray
+    >>>
+    >>> from ray.train.xgboost import XGBoostTrainer
+    >>> from ray.air.config import ScalingConfig
+    >>>
+    >>> train_dataset = ray.data.from_items(
+    ...     [{"x": x, "y": x + 1} for x in range(32)])
+    >>> trainer = XGBoostTrainer(
+    ...     label_column="y",
+    ...     params={"objective": "reg:squarederror"},
+    ...     scaling_config=ScalingConfig(num_workers=3),
+    ...     datasets={"train": train_dataset}
+    ... )
+    >>> result = trainer.fit() # doctest: +ELLIPSIS
+    == Status ...
```

Comment:
nit: I'm personally not familiar with doctesting, so this seemed kind of weird to me. Should we have consistency with  `stopper.py`? https://github.com/ray-project/ray/blob/e7d9c242afab6028a2dd87def73389969d1dbd70/python/ray/tune/stopper/stopper.py#L36-L37

Reply:
Hm last time this didn't pass for me but wemay have upgraded doctest in the meantime or so. I'll adjust and check